cinema/models.py

```diff
@@ -1,10 +1,38 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True, null=True)
```

Comment:
it can't be `null`. If at least two names would be `null` - then an error would occur as objects aren't unique